### DISCUSSION TITLE
Model for prints just so I can find them later

### DISCUSSION THREAD

**Discussion by CooperTrout** [frc5567/2023-Robot#123] [Team Posts]
Arduino Case
https://www.thingiverse.com/thing:99710
NavX-mxp
https://www.pdocs.kauailabs.com/navx-mxp/installation/creating-an-enclosure/

**Comment by coverbeek** [comment]
Hey Cooper,

The Arduino one looks good, but we'd need to be sure we can get our wires into the digital inputs/outputs that appear to be open on the cover. I don't want you to waste time/money on a print unless we're relatively sure of that. We can make custom wires if we need to.

For the Navx MXP - it is hard to tell from their plan if the MXP flat cable connector would work with the case - I don't see an obvious example in the link - can you tell by the files if there is space for the ribbon cable?

Thanks for researching!
-Chris
